### ISSUE TITLE
Add integration with Travis CI

### ISSUE THREAD

**Issue by daducci** [daducci/COMMIT#93]
IT would be nice to add integration with the [Travis IC platform](https://travis-ci.com).

**Comment by MarioOcampo** [comment]
Yes, it can be a good complement to our integration process (test the PRs before merging). 
We need to design/implement unit test, for each model or relevant section, let me think some options. Do you have any idea?

**Comment by nightwnvol** [comment]
Added Github Actions workflows:
- [x] Prebuilt wheels compilation
- [x] Automatic upload on PyPI
- [x] COMMIT execution tests (wiki example)

---

- #139
